installer/macos/create_pkg.sh

```diff
@@ -24,41 +22,30 @@ fatal() {
     exit 1
 }
 
-# Download the latest release of MassaStation app.
-download_massastation_app() {
-    curl -L https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_darwin-$ARCH -o $APP_BINARY_NAME || fatal "failed to download $APP_BINARY_NAME"
-    chmod +x $APP_BINARY_NAME || fatal "failed to chmod $APP_BINARY_NAME"
-}
-
-# Download the latest release of MassaStation server.
-build_massastation_server() {
-    go generate ../... || fatal "go generate failed for $SERVER_BINARY_NAME"
-    go build -o $SERVER_BINARY_NAME ../cmd/thyra-server || fatal "failed to build $SERVER_BINARY_NAME"
-    chmod +x $SERVER_BINARY_NAME || fatal "failed to chmod $SERVER_BINARY_NAME"
-}
-
-# Delete the build directory if it exists.
-clean() {
-    if [ -d $BUILD_DIR ]; then
-        rm -rf $BUILD_DIR || fatal "failed to delete $BUILD_DIR"
-    fi
+# Build MassaStation from source.
+build_massastation() {
+    go generate ../... || fatal "go generate failed for $MASSASTATION_BINARY_NAME"
+    export GOARCH=$ARCH
+    export CGO_ENABLED=1
+    fyne package -icon logo.png -name MassaStation -appID com.massalabs.massastation -src ../cmd/massastation || fatal "fyne package failed for $MASSASTATION_BINARY_NAME"
+    chmod +x $MASSASTATION_BINARY_NAME || fatal "failed to chmod $MASSASTATION_BINARY_NAME"
 }
 
 # Build the package using pkgbuild.
 package() {
-    pkgbuild --root $BUILD_DIR --identifier com.massalabs.massastation --version $PKGVERSION \
-        --scripts macos/scripts --install-location / massastation_$PKGVERSION\_$ARCH.pkg || fatal "failed to create package"
+    pkgbuild --component $MASSASTATION_BINARY_NAME --identifier com.massalabs.massastation --version $PKGVERSION \
+        --scripts macos/scripts --install-location /Applications massastation_$PKGVERSION\_$ARCH.pkg || fatal "failed to create package"
 }
 
-main() {
-    clean
+download_dependencies() {
+    go get fyne.io/fyne/v2@latest
```

Comment:
why not `go get` over `go install`?

Reply:
I followed the documentation: https://developer.fyne.io/started/#go-modules-for-go-116-or-newer
But you are right, we don't need that `go get` part.